src/apis/sourceimages.js

```diff
@@ -171,8 +177,15 @@ export default (state) => {
       }
     })
     .then((data) => {
+      var formData = {}
```

Comment:
please use `const`

Reply:
actually that and the following lines could be simplified to:
```js
const formData = Object.assign({}, metadata);
```

can you try that out? I only tried it in my browser console.

Reply:
you sure? I also add a '[0]' to the keys of all entries (since rokka expects that here) in that forEach loop

Reply:
ah, right.. somehow didn't look close enough 😅 
ok then that's fine if you change `var` to `const` (const, non-primitive (array, objects) variables can still be updated) and `const` has better scoping than `var`.